storage-provider/server/src/p2p/mod.rs

```diff
@@ -141,14 +146,20 @@ pub async fn run_register_node(
 ) -> Result<(), P2PError> {
     tracing::info!("Starting P2P register node");
     let tracker = TaskTracker::new();
-    let (swarm, rendezvous_point_address, rendezvous_point) = config.create_swarm()?;
+    // If TTL is not set, set it to 24 hours
+    let ttl = if config.registration_ttl.is_some() {
+        config.registration_ttl
+    } else {
+        Some(TTL_24_HOURS)
+    };
```

Comment:
This goes against `None` as a default and makes the code harder to follow since the default is set late. Just use `TTL_24_HOURS` as a default at the CLI level